app/assets/stylesheets/revised/pages/landing/_home.scss

```diff
@@ -12,18 +12,177 @@
 }
 
 .root-landing-banner-header {
-  background-image: image-url("landing_pages/header_bg_root-1080.jpg");
   width: 100%;
-  padding-top: 12 * $vertical-height;
-  padding-bottom: 6 * $vertical-height;
+  margin-bottom: 0;
+}
+
+.root-landing-banner-subheader {
+  width: 100%;
+  margin: 0;
+
+  .subheader-tile-link {
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .subheader-tile:first-of-kind {
+    padding-right: 5px;
+  }
+
+  .subheader-tile:last-of-kind {
+    padding-left: 5px;
+  }
+
+  .subheader-tile {
+    margin: 10px auto;
+    min-width: 275px;
+    padding-left: 5px;
+    padding-right: 5px;
+  }
+
+  .subheader-image {
+    height: 275px;
+  }
+
+  .subheader-image-caption {
+    background-color: black;
+    color: #fff;
+    font-weight: bold;
+    padding: 10px 15px;
+    text-decoration: none;
+  }
+}
+
+.center-navbar-signup-link-container {
+  display: inline-block;
+  margin: 20px auto 10px auto;
+  position: absolute;
+  text-align: center;
+  width: 75%;
+}
+
+.center-navbar-signup-link {
+	background-color: black;
+	border-radius: 4px;
+	border: 1px solid #3498DB;
+	width: 85px;
+  box-sizing: border-box;
```

Comment:
I believe border box is applied to everything via a psuedo `::before` element, and this isn't necessary